beta/project-griffon/troubleshoot-project-griffon.md

```diff
@@ -45,3 +45,32 @@ Please upgrade the Adobe Analytics and Assurance SDK extensions to the latest ve
 | --------------------------- | ------- | ------- |
 | Adobe Analytics             | > 2.4.0 | > 1.2.6 |
 | Assurance                   | > 1.0.0 | > 1.0.0 |
+
+
+
+### React Native MobileCore and AEPAssurance compatibility
+
+| AEP Assurance Version            | Mobile Core Version                                          | Install Instruction                                          |
+| -------------------------------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| react-native-aepassurance v2.x.x | [react-native-acpcore](https://www.npmjs.com/package/@adobe/react-native-acpcore) | npm install @adobe/react-native-aepassurance@^2.0.0 <br/>npm install @adobe/react-native-acpcore |
+| react-native-aepassurance v3.x.x | [react-native-aepcore](https://www.npmjs.com/package/@adobe/react-native-aepcore) | npm install @adobe/react-native-aepassurance@^3.0.0 <br/>npm install @adobe/react-native-aepcore |
+
+In case you are using `react-native-acpcore` with Assurance and the React Native application fails to build with one of the following error messages:
```

Comment:
This kind of sentence fragments and leads us down a path which doesn't get resolved.

I'd go with something like: 

"If you are using `react-native-acpcore` with AEPAssurance, the React Native application can fail to build with one of the following error messages:"